xml/cap_depl_eks.xml

```diff
@@ -370,151 +288,347 @@ Custom TCP Rule    TCP          4443            0.0.0.0/0       CAP WSS
 Custom TCP Rule    TCP          443             0.0.0.0/0       CAP HTTPS
 Custom TCP Rule    TCP          20000-20009     0.0.0.0/0       TCP Routing</screen>
  </sect1>
- <sect1 xml:id="sec.cap.eks-external-ips">
-  <title>Find your <literal>kube.external_ips</literal></title>
-
-  <para>
-   In your EC2 VM List, look up the private IP addresses of one of the nodes,
-   and note the address that is used in its private DNS, which looks like
-   <literal>ip-<replaceable>address</replaceable>.us-west-2.compute.internal</literal>.
-   Also note the public IP address, which you will need for the DOMAIN of the
-   cluster in your &productname; configuration file.
-  </para>
- </sect1>
- <sect1 xml:id="sec.cap.eks-deployment">
-  <title>Configuring and Deploying &productname;</title>
-
-  <para>
-   Follow the instructions in <xref linkend="sec.cap.storageclass-prod"/> to
-   deploy &productname;. When you create your
-   <filename>scf-config-values.yaml</filename> file, make the following
-   changes:
+ 
+ <sect1 xml:id="sec.cap.eks.dns-first">
+     <title>DNS Configuration</title>
+     <para>
+         Creation of the EKS load balancer is triggered by a setting in the &cap; 
+         deployment configuration file. This requires first mapping your cluster
+         domain to a worker node. If you are using the Amazon Route53 service 
+         for DNS, your domain and sub-domains are all CNAMEs.
+     </para>
+       <para>
+   For this first pre-deployment DNS configuration, look up the public and 
+   private IP addresses of one of the nodes in your virtual machine list. This 
+   node provides the initial access to your cluster, before the load balancer is
+   available. This private IP address is the <literal>external_ips:</literal> 
+   value in your deployment configuration file, which is called 
+   <filename>scf-config-values.yaml</filename> in this chapter (an example 
+   configuration file is provided in <xref linkend="sec.cap.eks-configuration"/>). 
+   Map the public IP address to your domain name.
   </para>
-
-  <itemizedlist>
-   <listitem>
-    <para>
-     Use <literal>overlay-xfs</literal> for
-     <literal>env.GARDEN_ROOTFS_DRIVER</literal>
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     use <literal>""</literal> for
-     <literal>env.GARDEN_APPARMOR_PROFILE</literal>
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     Set <literal>kube.storage_class.persistent</literal> and
-     <literal>kube.storage_class.shared</literal> to <literal>gp2</literal>
-    </para>
-   </listitem>
-  </itemizedlist>
-
-  <para>
-   The following roles require <literal>SYS_RESOURCE</literal> capabilities:
+     
```

Comment:
I'll let @troytop confirm but I don't believe this is needed any more. With `services.loadbalanced=true` we should not need to deal with external_ips. We just need to wait until the kube public services get their hostnames on the ELB and once those hostnames are available create appropriate CNAME entries for the values specified in DOMAIN, UAA_HOST etc., e.g.,  UAA_HOST should now be a CNAME pointing to the hostname for the uaa-public kube service.

Reply:
Actually, right… if we did the extra port fix ahead of time, everything including TCP routing should come up fine; the down side here is that we will briefly expose the healthcheck port to the internet (but it shouldn't be very interesting).

Reply:
This is my biggest point of confusion. It would be lovely to not have to do the funky DNS shuffle.